src/base/components/widgets/Login.js

```diff
@@ -1,70 +1,70 @@
-import React from 'react';
-import Details from './Details.js';
-import { getServiceUrl, getDomain } from '../../servicelocator.js';
-import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+// import React from 'react';
+// import Details from './Details.js';
+// import { getServiceUrl, getDomain } from '../../servicelocator.js';
+// import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
 
-const domain = getDomain();
+// const domain = getDomain();
 
-const LoginButton = ({ onLogin }) => (
-  <a href="#" className="btn btn-just-icon btn-simple btn-default btn-sm btn-flat pull-right">
-    <i className="material-icons dp_big">account_circle</i>
-    Login
-  </a>
-);
+// const LoginButton = ({ onLogin }) => (
+//   <a href="#" className="btn btn-just-icon btn-simple btn-default btn-sm btn-flat pull-right">
+//     <i className="material-icons dp_big">account_circle</i>
+//     Login
+//   </a>
+// );
```

Comment:
Let's left the login button. We will use it later.

Reply:
Ok sure.

I was thinking that it is not useful anymore.